example/s3select_example.cpp

```diff
@@ -382,6 +390,7 @@ int process_json_query(const char* input_query,const char* fname)
 
   auto object_sz = boost::filesystem::file_size(fname);
   json_object json_query_processor(&s3select_syntax);
+  m_s3_json_object.set_json_query(&s3select_syntax, json);
```

Comment:
there is no need for `m_s3_json_object`.
we already have `json_query_processor`
please note that calling to `set_json_query` causes duplicate initialization.